### PR TITLE
feat: fix wallet address tooltip strings

### PR DESCRIPTION
**Pull request by tinaszheng**

### CODE REVIEW

File: scripts/generated/tooltips.ts [hunk 1]
```diff
@@ -35,7 +35,8 @@ export const TOOLTIP_STRING_KEYS = {
 
     // GENERAL
 
-    DYDX_ADDRESS_BODY: 'TOOLTIPS.GENERAL.DYDX_ADDRESS.BODY',
```
Comment (tyleroooo) [inline scripts/generated/tooltips.ts:38]: technically we need to ensure mobile isn't using this since this is a breaking change

**Review by tyleroooo — APPROVED** [review]
just make sure we don't break mobile since they fetch this live

File: scripts/generated/tooltips.ts (hunk 1 above)
Comment (tinaszheng) [reply scripts/generated/tooltips.ts:38]: ooo maybe i should just add a new one?

Comment (tinaszheng) [reply scripts/generated/tooltips.ts:38]: https://github.com/search?q=org%3Adydxprotocol+DYDX_ADDRESS_BODY&type=code doesnt looook like theyre using it but i asked in the channel

**Pull request merged**